rasa/core/validator.py

```diff
@@ -199,3 +199,14 @@ def verify_all(self, ignore_warnings: bool = True) -> bool:
         logger.info("Validating utterances...")
         stories_are_valid = self.verify_utterances_in_stories(ignore_warnings)
         return intents_are_valid and stories_are_valid and there_is_no_duplication
+
+    def verify_domain_is_empty(self) -> bool:
+        """Checks if the domain is empty."""
+
+        return self.domain.is_empty()
+
+    def verify_domain(self) -> bool:
+        """ Runs the validations on domain."""
+
+        is_valid = not self.verify_domain_is_empty()
+        return is_valid
```

Comment:
To be clear, the way this is working is that if the domain is invalid, the importer won't return anything, therefore `self.domain.is_empty()` is true?

If so, I think it makes more sense to merge these 2 functions(one just calls the other anyway) into one called e.g. `verify_domain_validity`

Reply:
Yeah, it makes total sense!